Packs/SecureWorks/Integrations/TaegisXDR/README.md

```diff
@@ -59,14 +59,18 @@ After you successfully execute a command, a DBot message appears in the War Room
 
 #### Input
 
-| **Argument Name** | **Description** | **Required** |
-| --- | --- | --- |
-| ids | A list of alerts by IDs | True |
+| **Argument Name** | **Description** | Default | **Required** |
+| --- | --- | --- | --- |
+| ids | A list of alerts by IDs | `936c1cc1-db8f-430c-837c-1c914fcca35a` | False |
+| limit | Number of results to when `ids` is not defined | `10` | False |
+| offset | The result to start from when `ids` is not defined | `0` | False |
+| cql_query | The query to utilize when searching for Alerts | `from alert severity >= 0.6 and status='OPEN'` | False |
 
 #### Command Examples
 
 ```
 !taegis-fetch-alerts ids=`["6594e97f-a898-5b28-82b2-ea03293cdaa1"]`
+!taegis-fetch-investigation-alerts id=936c1cc1-db8f-430c-837c-1c914fcca35a
```

Comment:
If these are the same command, maybe we should consider deprecating the first one?

Reply:
I think this is a typo, I accidentally added the `taegis-fetch-investigation-alerts` line there. Technically, they're not the same command (1 is for pulling a specific alert while the other is for any alerts related to a specific investigation).

We could potentially remove `taegis-fetch-investigation-alerts` command altogether as the alerts can be fetched from the `fetch-investigation` query as well, however this command still works and I'd rather not remove this in case it's in use by any users.

I'll remove this line from the README though as that command is actually described further down in the doc.